aspnetcore/migration/31-to-50.md

```diff
@@ -120,3 +120,24 @@ As part of the move to ".NET" as the product name, the Docker images moved from
 ## Review breaking changes
 
 For breaking changes from .NET Core 3.1 to .NET 5.0, see [Breaking changes for migration from version 3.1 to 5.0](/dotnet/core/compatibility/3.1-5.0). ASP.NET Core and Entity Framework Core are also included in the list.
+
+## Changes to model binding in ASP.NET Core MVC and Razor Pages
```

Comment:
I don't think these qualify as breaking changes but it is useful to document these in some place and point users to if they ask.